modules/appeals_api/app/controllers/concerns/appeals_api/higher_level_reviews/pdf_format_validation.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'appeals_api/higher_level_review/phone'
+
+module AppealsApi
+  module HigherLevelReviews
+    class PdfFormatValidation
+      def initialize(request_body, request_headers, version_validator:)
+        @request_body = request_body
+        @request_headers = request_headers
+        @version_validator = version_validator.new(request_body, request_headers)
+      end
+
+      def validate!
+        return error(422, veteran_phone.too_long_error_message) if veteran_phone.too_long?
```

Comment:
I don't think that HLR v2 has the same character limits for phone number as v1 did, so I don't really think this shared utility class is necessary.

I think we could remove this class and just let each version validator get called directly from each controller. Even if there's a bit of duplication (in the case that the phone number validation _was_ the same across both versions), I think that separating out what's valid for each version is fine and somewhat more clear - I won't have to remember which layer handled which validation, I'd just look at the validator for that version. Thoughts?